test/csuite/timestamp_abort/main.c

```diff
@@ -143,70 +145,67 @@ usage(void)
     exit(EXIT_FAILURE);
 }
 
+/*
+ * maximum_committed_ts --
+ *     Return the largest timestamp that's no longer in use.
+ */
+static uint64_t
+maximum_committed_ts(void)
```

Comment:
Can we move this to test_util so we don't need to duplicate this everywhere?

Reply:
I like the idea but I'm not sure there's a clean way to implement it since it looks inside the `THREAD_DATA` structure. Do you think it's reasonable to either move this data structure into `test_util.h` as well, or change the test logic to keep the commit timestamps in an array that can be passed in? Or perhaps something else would be better.

Reply:
change the test logic to keep the commit timestamps in an array that can be passed in

I think this is good enough.

Reply:
Thanks, this is now fixed in 5f2eb51. I also re-ran both `tiered_abort` and `timestamp_abort` for a bit, seems to just as robust.